SketchAssistant/SketchAssistantWPF/FileImporter.cs

```diff
@@ -65,21 +65,21 @@ public class FileImporter
             String[] size = allLines[1].Split('x');
             width = Convert.ToInt32(size[0]);
             height = Convert.ToInt32(size[1]);
-            return (width, height);
+            return new Tuple<int, int>(width, height);
         }
 
         /// <summary>
         /// parses all line entries of an input file in .isad format
         /// </summary>
         /// <param name="allLines">the input file as an array of lines</param>
         /// <returns>the parsed picture as a list of lines</returns>
-        private List<Line> ParseISADBody(String[] allLines, int width, int height)
+        private List<InternalLine> ParseISADBody(String[] allLines, int width, int height)
         {
 
             String lineStartString = "line";
             String lineEndString = "endline";
 
-            List<Line> drawing = new List<Line>();
+            List<InternalLine> drawing = new List<InternalLine>();
 
             //number of the line currently being parsed, enumeration starting at 0, body starts at the third line, therefore lin number 2
             int i = 2;
```

Comment:
vlt umbenennen in currentLineNumber oder ähnliches, an sich aber mit dem Kommentar verständlich